testsuite/integration/microprofile-tck/jwt/tck-base-suite.xml

```diff
@@ -40,6 +40,9 @@
         </groups>
         <classes>
             <class name="org.eclipse.microprofile.jwt.tck.util.TokenUtilsTest" />
+            <class name="org.eclipse.microprofile.jwt.tck.util.TokenUtilsEncryptTest" />
```

Comment:
@darranl Out of curiosity why are you listing all these classes here? It does not seem to be needed (to this extent) for the other specs

Reply:
I believe this approach was from the instructions in MicroProfile JWT at the time 1.1 was initially added to identify the mandatory set of tests.  I would need to check but there may have been additional test on top of this mandatory set.